### PR TITLE
bugfix/fix padding margin

### PR DESCRIPTION
**Pull request by LeVHoangduc**

### CODE REVIEW

**Comment by LeVHoangduc** [comment]
@hungnguyenhuu  @lenguyen-agilityio  Please help me to check PR and merge it

File: practice-one/src/index.html [hunk 1]
```diff
@@ -139,7 +139,7 @@ <h1 class="titleElement">How VanGoux works</h1>
                 <img src="assets/images/process/upload.png" alt="Icon Upload" />
               </div>
             </div>
-            <p class="step__content">
+            <p class="">
```
Comment (hungnguyenhuu) [inline practice-one/src/index.html:142]: Empty class?

File: practice-one/src/index.html [hunk 2]
```diff
@@ -164,9 +164,7 @@ <h1 class="titleElement">How VanGoux works</h1>
                 />
               </div>
             </div>
-            <p class="step__content">
-              You get a publicly shareable link for your website
-            </p>
+            <p class="">You get a publicly shareable link for your website</p>
```
Comment (hungnguyenhuu) [inline practice-one/src/index.html:167]: Empty class?

File: practice-one/src/index.html (hunk 1 above)
Comment (hungnguyenhuu) [reply practice-one/src/index.html:142]: Check the similar cases

Comment (LeVHoangduc) [reply practice-one/src/index.html:142]: @hungnguyenhuu I made a mistake when handling the conflict. i updated it

File: practice-one/src/index.html (hunk 2 above)
Comment (LeVHoangduc) [reply practice-one/src/index.html:167]: @hungnguyenhuu I made a mistake when handling the conflict. i updated it

File: practice-one/src/index.html [hunk 3]
```diff
@@ -277,40 +271,30 @@ <h3 class="text">Complete code freedom</h3>
...
-            <p class="user__content">
-              Auto-generate boilerplate components and design systems
-            </p>
+            <h2 class="user__title">React developers</h2>
+            <p>Auto-generate boilerplate components and design systems</p>
           </div>
           <div class="user">
-            <h3 class="user__title">Design agencies</h3>
-            <p class="user__content">
+            <h2 class="user__title">Design agencies</h2>
```
Comment (lenguyen-agilityio) [inline practice-one/src/index.html:286]: @LeVHoangduc 
Why change it to h2? It mean the font size change to 30px right? As I checked design, the font size should be 20px

Comment (LeVHoangduc) [reply practice-one/src/index.html]: This is the error when i handle merge conflict. I updated it again

**Pull request merged**